src/Compilers/CSharp/Portable/Symbols/Source/FieldSymbolWithAttributesAndModifiers.cs

```diff
@@ -395,6 +395,8 @@ internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, r
                     DeclaringCompilation.SynthesizeTupleNamesAttribute(type.TypeSymbol));
             }
 
+            // PROTOTYPE(NullableReferenceTypes): type.ReportAnnotatedUnconstrainedTypeParameterIfAny()
```

Comment:
Why can't we do this today?

Reply:
We should be able to do these today, but these were cases where we may need to pass through a `DiagnosticBag`. It seemed more important to get the general approach for now rather than covering all the corner cases.

---
In reply to: [205633140](https://github.com/dotnet/roslyn/pull/28804#discussion_r205633140) [](ancestors = 205633140)